design/space_template.go

```diff
@@ -0,0 +1,101 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var spaceTemplate = a.Type("SpaceTemplate", func() {
+	a.Description(`JSONAPI store for the data of a space template. See also http://jsonapi.org/format/#document-resource-object`)
+	a.Attribute("type", d.String, func() {
+		a.Enum("spacetemplates")
+	})
+	a.Attribute("id", d.UUID, "ID of space template", func() {
+		a.Example("40bbdd3d-8b5d-4fd6-ac90-7236b669af04")
+	})
+	a.Attribute("attributes", spaceTemplateAttributes)
+	a.Attribute("links", genericLinks)
+	a.Required("type", "attributes")
+})
+
+var spaceTemplateAttributes = a.Type("SpaceTemplateAttributes", func() {
+	a.Description(`JSONAPI store for all the "attributes" of a space template. See also see http://jsonapi.org/format/#document-resource-object-attributes`)
+	a.Attribute("name", d.String, "name of the space template", func() {
+		a.Example("Issue Tracking")
+		a.MinLength(1)
```

Comment:
I believe we want to follow the same restrictions than with the space names. Can you re-use what I added to the space template ?